src/submission/template-api.ts

```diff
@@ -0,0 +1,49 @@
+import * as manager from "../lectern-client/schema-manager";
+import { Request, Response } from "express";
+import { loggerFor } from "../logger";
+import { SchemaDefinition, ValueType } from "../lectern-client/schema-entities";
+import { setStatus, Status } from "../app-health";
+
+const L = loggerFor(__filename);
+
+export const get = async (req: Request, res: Response) => {
+  const schemaName: string = req.params.schemaName;
+  const schemasDictionary = manager.instance().getCurrent();
+  if (!schemasDictionary) {
+    setStatus("schema", { status: Status.ERROR });
```

Comment:
I don't think this is a right place for this, I prefer we keep the status update in the bootstrap logic, I also think it will already be set to error at this point